README.md

```diff
@@ -124,3 +128,71 @@ interpreting them with HIBA.
 libhiba provides convenience functions to parse and extract HIBA extensions
 directly from a certificate, but it does not verify the certificate. It assumes
 the certificate was validated by a prior stage.
+
+### Example CA
+
+Once you have completed the
+[compilation instructions](https://github.com/google/hiba/blob/main/INSTALL.md#compilation)
+you will have a `hiba-gen` and `hiba-ca.sh` in your hiba directory. You can now
+create CA keys, grants, identities, grant eligibility, and sign host and user
+certificates with these extensions. The following assumes you are in your hiba
+directory.
+
+1. Choose a directory for the CA structure.
```

Comment:
I'll work on the history revision - my git knowledge is limited so I'll have to go look that up. Changed everything to remove -d and instead use ~/.hiba-ca.

Reply:
You can run `git rebase -i HEAD~N` with N the number of revisions you want to revisit.

Reply:
I did that, and then learned I need to prefix each of the commits with a "squash" word in the editor - so `git log` looks good locally, but I'm not able to push to origin because "Updates were rejected because the tip of your current branch is behind its remote counterpart". The fast forward makes me nervous, and now if I try to git pull I get a hint about divergent branches and setting git config stuff - so I set the strategy to merge and re-pulled down, which just gave me back my prior commits and added a new one for the merge - so I assume I am missing some --just-replace-origin-plz command.

Reply:
The answer here is git push -f origin <branch> 

The -f is a problem on the main branch (or any public branch), but it is perfectly fine in your dev branches.
If you worry about an accidental bad push, don't: the main branch is protected against direct pushes for everyone (including maintainers), and only can be updated via the review process.

Reply:
done.